apps/passport-client/src/dispatch.ts

```diff
@@ -741,8 +756,7 @@ async function addSubscription(
   await saveSubscriptions(state.subscriptions);
   update({
     subscriptions: state.subscriptions,
-    loadedIssuedPCDs: false,
-    loadingIssuedPCDs: false
```

Comment:
why did you remove one of these? i suspect the field `loadingIssuedPCDs` should be removed altogether?

Reply:
This was a mistake.  I started trying to remove loadingIssuedPCDs, then aborted when I found that it was actually used to update UI, which I didn't want to mess with right now.

I didn't realize until now that loading issued PCDs can happen more than once.  This code is sufficient to trigger it to happen again if it's finished, but not if it's currently in progress.  That's not changing in this PR, though, and I don't see an obvious solution, so I'm not going to try to change it.

Given loadingIssuedPCDs is now only for UI, I don't think there's any need to force it to false here, though.  I'm not sure that was ever right, but I'm going to remove it.  It's setting `loadingIssuedPCD` to false which will force the fetch to happen again.